### PR TITLE
First two TESS beginner tutorials.

### PR DESCRIPTION
**Pull request by scfleming**
These from a fresh fork, so there should just be a single commit.

### CODE REVIEW

File: notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+    "calibrated_fluxes[4,:,:].shape"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Show the calibrated pixel values.\n",
+    "\n",
+    "Let's display the calibrated fluxes for one of the cadences in our target pixel file.  For now, we'll pick the fifth cadence (index number 4).  You could of course decide to look at multiple cadences by looping over each index in the first dimension of the structure (e.g., loop over calibrated_fluxes[i,:,:]).  You could even make an animated .gif of the fluxes and see how they change from cadence to cadence.  To get you started though, let's just display one of the cadence's flux values."
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb]: Suggest calibrated_fluxes[4,:,:] -> \`calibrated_fluxes[4,:,:]\` to set it off from the surrounding text.

File: notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+    "fig, ax = plt.subplots()\n",
+    "# Display the calibrated fluxes as an image for the fifth cadence.\n",
+    "cax = ax.imshow(calibrated_fluxes[4,:,:], cmap=plt.cm.YlGnBu_r,\n",
+    "                origin=\"lower\")\n",
+    "# Let's define a title for the figure.\n",
+    "fig.suptitle(\"WASP-126 b Calibrated Fluxes - Sector 1, Fifth Cadence\")\n",
+    "# Add a color bar.\n",
+    "cbar = fig.colorbar(cax)\n",
+    "plt.show()"
+   ]
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb]: Suggest adding newlines before each comment line to improve readability of the cell.

File: notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Start figure and axis.\n",
+    "fig, ax = plt.subplots()\n",
+    "# Display the calibrated fluxes as an image for the fifth cadence.\n",
+    "cax = ax.imshow(calibrated_fluxes[4,:,:], cmap=plt.cm.YlGnBu_r,\n",
+    "                origin=\"lower\")\n",
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb]: Line break unnecessary?

File: notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb [hunk 4]
```diff
@@ -0,0 +1,354 @@
...
+    "# Start figure and axis.\n",
+    "fig, ax = plt.subplots()\n",
+    "# Display the pixels as an image.\n",
+    "cax = ax.imshow(aperture, cmap=plt.cm.YlGnBu_r, origin=\"lower\")\n",
+    "# Add a color bar.\n",
+    "cbar = fig.colorbar(cax)\n",
+    "# Add a title to the plot.\n",
+    "fig.suptitle(\"WASP-126 b Aperture - Sector 1\")\n",
+    "plt.show()"
+   ]
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb]: Suggest adding newlines before each comment line to improve readability of the cell.

File: notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb [hunk 5]
```diff
@@ -0,0 +1,354 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Break down a pixel value of 267 (yellow pixels displayed above) into its\n",
+    "# constituent bits.\n",
+    "bits = np.binary_repr(267)\n",
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb]: Unnecessary line break?

File: notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb [hunk 6]
```diff
@@ -0,0 +1,354 @@
...
+    "    for j, pix in enumerate(row):\n",
+    "        # Get the bitmask as a string of zeros and ones.\n",
+    "        this_bitmask = np.binary_repr(pix)\n",
+    "        # Is the bitmask value of 2 set?  If so, the 2nd integer from the\n",
+    "        # right should be set to 1.  We'll use Python negative indexes to\n",
+    "        # access the second integer from the right.\n",
+    "        if this_bitmask[-2] == '1':\n",
+    "            # Then record this pixel as having a bitmask value of 2 set.\n",
+    "            bitmask2_set[i,j] = 1"
+   ]
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb]: Suggest adding newlines before each comment line (or group of comment lines) to improve readability of the cell.

File: notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb [hunk 7]
```diff
@@ -0,0 +1,354 @@
...
+    "fig, ax = plt.subplots()\n",
+    "# Display, as an image, the 11x11 table that records the bitmask value of\n",
+    "# 2 being set.\n",
+    "cax = ax.imshow(bitmask2_set, cmap=plt.cm.YlGnBu_r, origin=\"lower\")\n",
+    "# Add a color bar.\n",
+    "cbar = fig.colorbar(cax)\n",
+    "# Add a title to the plot.\n",
+    "fig.suptitle(\"WASP-126 b Aperture - Sector 1 - Pixels Used In Phot. Ap.\")\n",
+    "plt.show()"
+   ]
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb]: Suggest adding newlines before each comment line to improve readability of the cell.

File: notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb [hunk 8]
```diff
@@ -0,0 +1,354 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Start figure and axis.\n",
+    "fig, ax = plt.subplots()\n",
+    "# Display, as an image, the 11x11 table that records the bitmask value of\n",
+    "# 2 being set.\n",
+    "cax = ax.imshow(bitmask2_set, cmap=plt.cm.YlGnBu_r, origin=\"lower\")\n",
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb]: Unnecessary newline?

File: notebooks/MAST/TESS/beginner_how_to_use_lc/beginner_how_to_use_lc.ipynb [hunk 9]
```diff
@@ -0,0 +1,323 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Obtaining The Light Curve File\n",
+    "\n",
+    "We will read the light curve file from Sector 1 using the MAST URL location.  So that we can get started with understanding the file contents without reviewing how to automatically search for and retrieve TESS files, we won't show how to search and retrieve TESS light curve files in this tutorial.  Consult other TESS tutorials for much more information on how to search for TESS data using MAST services."
+   ]
```
[the remark this answers is not in the document]
Comment (scfleming) [reply notebooks/MAST/TESS/beginner_how_to_use_lc/beginner_how_to_use_lc.ipynb]: Adding a link to the Archive Manual Tutorials page.

File: notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb [hunk 10]
```diff
@@ -0,0 +1,354 @@
...
+   "source": [
+    "print(type(calibrated_fluxes))\n",
+    "print(calibrated_fluxes.shape)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "So these data are stored as a numpy array.  The first dimension is the number of cadences, followed by the size of the readout.  Thus, accessing calibrated_fluxes[4,:,:] should return to us the 11x11 grid of pixels from the **fifth** cadence (remember Python is zero-indexed, so the first cadence is stored in the zeroth index, the second in the first index, etc.)"
```
[the remark this answers is not in the document]
Comment (scfleming) [reply notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb]: Cool!  Didn't know about that markdown.

**Comment by eteq** [comment]
@scfleming @ceb8 - this a minor thing, but: the requirements aren't actually *exact versions*, right (i.e. ``astropy==3.0.4``)? Instead, I'm guessing the versions listed are "known good"?  In that case instead it's probably better to do e.g. ``astropy>=3.0.4``, as then a user can just do `pip install -r requirements.txt`.

(If that's right, I can just update the requirements files directly in master - I just wanted to double-check before doing so.)

**Comment by scfleming** [comment]
Yes they are min.

**Pull request merged**